services/storage/src/simcore_service_storage/db_access_layer.py

```diff
@@ -154,6 +155,24 @@ async def list_projects_access_rights(
     return projects_access_rights
 
 
+def _get_project_access_rights_stmt(user_id: UserID, project_id: ProjectID):
+    user_gids = (
+        sa.select(sa.func.array_agg(sa.cast(user_to_groups.c.gid, sa.String)))
+        .where(user_to_groups.c.uid == f"{user_id}")
+        .group_by(user_to_groups.c.uid)
+    )
+
+    return sa.select(projects.c.prj_owner, projects.c.access_rights).where(
+        (projects.c.uuid == f"{project_id}")
+        & (
+            (projects.c.prj_owner == f"{user_id}")
+            | sa.func.jsonb_exists_any(
+                projects.c.access_rights, user_gids.scalar_subquery()
```

Comment:
project.c.access_rights is already redundant and should not be used after this refactoring: https://github.com/ITISFoundation/osparc-simcore/pull/6060 . (I have kept the column there to be extra safe and if somethings goes wrong we have still the data/structure there as it was before, my plan was to remove it after it goes to production in next sprint). Now we need to use `project_to_groups`, ex.: 

```
            access_rights_subquery = (
                sa.select(
                    project_to_groups.c.project_uuid,
                    sa.func.jsonb_object_agg(
                        project_to_groups.c.gid,
                        sa.func.jsonb_build_object(
                            "read",
                            project_to_groups.c.read,
                            "write",
                            project_to_groups.c.write,
                            "delete",
                            project_to_groups.c.delete,
                        ),
                    ).label("access_rights"),
                ).group_by(project_to_groups.c.project_uuid)
            ).subquery("access_rights_subquery")
```
as used in the simcore_service_webserver/projects/db.py

Yes indeed this is bug from my side I didn't notice the usage of project access rights with these `sa.DDL` statements here. I double check and this should be the last place where it is used.

Reply:
@matusdrobuliak66 takes over and will contribute with some changes to the PR